csrankings-x.csv

```diff
@@ -218,7 +218,7 @@ Xiaopeng Hong,Harbin Institute of Technology,http://homepage.hit.edu.cn/hongxiao
 Xiaoping Chen,USTC,http://dsxt.ustc.edu.cn/zj_ywjs.asp?zzid=471,7_dDukkAAAAJ
 Xiaoping Jia,DePaul University,http://condor.depaul.edu/xjia,NOSCHOLARPAGE
 Xiaoping Zhang,Tsinghua University,http://www.cs.tsinghua.edu.cn/publish/csen/4623/2010/20101225000806223958364/20101225000806223958364_.html,p82_4bgAAAAJ
-Xiaoqing Liu,University of Arkansas,https://computer-science-and-computer-engineering.uark.edu/directory/index/uid/frankliu/name/Xiaoqing+Liu,NOSCHOLARPAGE
+Xiaoqing Frank Liu,SIU Carbondale,https://engineering.siu.edu/elec/faculty-staff/faculty/liu.php,8ac8nKkAAAAJ

```

Comment:
Also needs alias entry `Xiaoqing (Frank) Liu`. However, in Computer Engineering, not in CS; please explain how they meet the stated inclusion criteria.

Reply:
Dr. Xiaoqing Frank Liu is also listed as a faculty in the Department of Computer Science - https://cs.siu.edu/faculty-staff/continuing_faculty.php

Reply:
I am also not sure of the term "Continuing Faculty". 

However, according to this [website](https://policies.siu.edu/personnel-policies/chapter2/ch2-faps/typeappt.php), 

Continuing appointment: A continuing appointment, either faculty or administrative/ professional staff, is one which is automatically renewed each year unless the appointee is given notice as specified in the appropriate personnel policies

Reply:
I expect that's not what it means in this context, but as it is unclear that this person meets the inclusion criteria, please remove them from this PR.